psi4/src/psi4/libsapt_solver/fdds_disp.cc

```diff
@@ -155,23 +156,84 @@ FDDS_Dispersion::FDDS_Dispersion(std::shared_ptr<BasisSet> primary, std::shared_
     // Build DFHelper
     dfh_ = std::make_shared<DFHelper>(primary_, auxiliary_);
     dfh_->set_memory(doubles);
-    dfh_->set_method("DIRECT_iaQ");
+    dfh_->set_method("DIRECT_iaQ"); // STORE is meaningless, no metric contraction needed
+    if (is_hybrid_) {
+        dfh_->set_method("DIRECT");
+    }
     dfh_->set_nthreads(nthread);
     dfh_->set_metric_pow(0.0);
     dfh_->initialize();
+    dfh_->print_header();
 
     // Define spaces
-    dfh_->add_space("i", Cstack_vec[0]);
-    dfh_->add_space("a", Cstack_vec[1]);
-    dfh_->add_space("j", Cstack_vec[2]);
-    dfh_->add_space("b", Cstack_vec[3]);
+    dfh_->add_space("a", Cstack_vec[0]);
+    dfh_->add_space("r", Cstack_vec[1]);
+    dfh_->add_space("b", Cstack_vec[2]);
+    dfh_->add_space("s", Cstack_vec[3]);
```

Comment:
what's the thought behind the index renaming? will this create inconsistency with any other portions of code?

Reply:
This is just to make them to be same with my notes with equations. For consistency, I don't think  mostother portions of the code would explicitly include any MO integrals; Electrostatics, exchange and induction equations are represented in AO. Exchange-dispersion is redirected to Ed Hohenstein's SAPT code which uses generalized 2e integrals, which is very different from any of the other part of code in notations anyway.